mldsa/sign.c

```diff
@@ -67,10 +59,28 @@ int crypto_sign_keypair(uint8_t *pk, uint8_t *sk)
   /* Compute H(rho, t1) and write secret key */
   shake256(tr, TRBYTES, pk, CRYPTO_PUBLICKEYBYTES);
   pack_sk(sk, rho, tr, key, &t0, &s1, &s2);
-
   return 0;
 }
 
+/*************************************************
+ * Name:        crypto_sign_keypair
+ *
+ * Description: Generates public and private key.
+ *
+ * Arguments:   - uint8_t *pk: pointer to output public key (allocated
+ *                             array of CRYPTO_PUBLICKEYBYTES bytes)
+ *              - uint8_t *sk: pointer to output private key (allocated
+ *                             array of CRYPTO_SECRETKEYBYTES bytes)
+ *
+ * Returns 0 (success)
+ **************************************************/
+int crypto_sign_keypair(uint8_t *pk, uint8_t *sk)
+{
+  uint8_t seed[SEEDBYTES];
+  randombytes(seed, SEEDBYTES);
+  return crypto_sign_keypair_internal(pk, sk, seed);
+}
+
 /*************************************************
```

Comment:
ahh, I didn't think this PR would be implementing the internal/external split as well. Also in https://github.com/pq-code-package/mldsa-native/pull/42

Reply:
I rabased on top of your changes.